index.js

```diff
@@ -89,20 +89,22 @@ async function loadCurrentConfig(setupConfig) {
 async function updateEslintConfig(setupConfig, config, currentConfig) {
   const generatedConfig = setupConfig.createEslintConfig(config);
   const configPath = getConfigPath(setupConfig);
+  const mergedConfigs = mergeConfigs(currentConfig, generatedConfig);
 
-  await writeFile(
-    configPath,
-    `---\n${yaml.safeDump(mergeConfigs(currentConfig, generatedConfig))}`
-  );
+  if (configPath.endsWith('.js')) {
+    await writeFile(configPath, mergedConfigs);
```

Comment:
This should probably be something like
```suggestion
    await writeFile(configPath, `module.exports = ${JSON.stringify(mergedConfigs, null, 2)}`);
```